### ISSUE TITLE
Network Graph API Implementation

### ISSUE THREAD

**Issue by amindadgar** [RnDAO/tc-serverComm#71]
This issue is created to shared to show how to output the Neo4J graph in order to prepare the data for front end developers.

## **tldr**:
- Using a cypher query, query the Neo4J database.
- Using some array manipulation and json object (or python dictionary) manipulation produce the results for the front-end.
   - The retrieved data is in the format of (it's real data)
```
{"type":"relationship","id":"16","label":"INTERACTED","properties":{"createdAt":"2023-04-04T09:10:50.545220","dates":["2023-02-07T00:00:00","2023-02-08T00:00:00","2023-02-09T00:00:00","2023-02-10T00:00:00","2023-02-11T00:00:00","2023-02-12T00:00:00","2023-02-07T00:00:00","2023-02-08T00:00:00","2023-02-09T00:00:00","2023-02-10T00:00:00","2023-02-11T00:00:00","2023-02-12T00:00:00"],"weights":[2.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0]},"start":{"id":"2","labels":["DiscordAccount"],"properties":{"createdAt":"2023-04-04T09:10:50.545220","username":"katerinabc#6667"}},"end":{"id":"4","labels":["DiscordAccount"],"properties":{"createdAt":"2023-04-04T09:10:50.545220","username":"sepehr#3795"}}}
 ```
   - The format that should be given to the front end is (it's mock data)
```
graph_data = [
  ['user1', 'user2'],
  ['user2', 'user3'],
  ['user3', 'user1']
]
``` 

## Detailed Explanation
A very complete and detailed explanation is written in the thread `Discord Neo4J Data Visualize` of `tc-front-end`: https://discord.com/channels/915914985140531240/1093442940745035828/1093442946793230367

Please if you had any questions or issues share here or in discord thread.
